Dockerfiles/Dockerfile-aws

```diff
@@ -18,29 +21,22 @@ RUN set -eux \
 	&& find /usr/lib/ -name '*.pyc' -print0 | xargs -0 -n1 rm -rf
 
 # aws-iam-authemticator (copied to final image)
+ARG PLATFORM_SHORT
+ARG IAM_AUTH_VERSION=1.19.6
 RUN set -eux \
-	&& git clone https://github.com/kubernetes-sigs/aws-iam-authenticator /tmp/aws-iam-authenticator \
-	&& cd /tmp/aws-iam-authenticator \
-	&& IAM_AUTH_VERSION="$( git tag --sort=-creatordate \
-		| grep -E 'v[0-9]+\.[0-9]+\.[0-9]+$' \
-		| head -1 \
-		| sed 's/^v//g' \
-	)" \
-	&& curl -sS -L -b cookies.txt -c cookies.txt \
+	&& curl -sS -L \
 		-o /usr/bin/aws-iam-authenticator \
-		-H "User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/44.0.2403.89 Safari/537.36" \
-		"https://github.com/kubernetes-sigs/aws-iam-authenticator/releases/download/v${IAM_AUTH_VERSION}/aws-iam-authenticator_${IAM_AUTH_VERSION}_linux_amd64" \
+		"https://amazon-eks.s3.us-west-2.amazonaws.com/${IAM_AUTH_VERSION}/2021-01-05/bin/linux/${PLATFORM_SHORT}/aws-iam-authenticator" \
```

Comment:
arm64 binary is not available from https://github.com/kubernetes-sigs/aws-iam-authenticator/releases/tag/v0.5.3

This is using URL from https://docs.aws.amazon.com/eks/latest/userguide/install-aws-iam-authenticator.html and changed amd64 to arm64 in URL.

Need to find a better fix for this - everywhere else it uses latest release instead of hardcoding version...

Reply:
Someone else has requested arm64 binaries already https://github.com/kubernetes-sigs/aws-iam-authenticator/issues/363

Reply:
Maybe can get them from here https://distro.eks.amazonaws.com/#release-channels .. the version dependency manifests has URL to current aws-iam-authenticator ..

Reply:
Latest aws-iam-authenticator now has arm64 release binaries https://github.com/kubernetes-sigs/aws-iam-authenticator/releases/